src/test/java/dev/jbang/BaseTest.java

```diff
@@ -0,0 +1,23 @@
+package dev.jbang;
+
+import java.io.IOException;
+
+import org.junit.Rule;
+import org.junit.contrib.java.lang.system.EnvironmentVariables;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.rules.TemporaryFolder;
+
+public abstract class BaseTest {
+
+	@BeforeEach
```

Comment:
This PR is nice @quintesse . It also avoided many duplications.

Would be better to use BeforeAll instead `BeforeEach`?

Reply:
Then it would only do it for each test _file_, I actually want to do it for _each test_ so one test can't affect the result of another.